src/hotspot/os_cpu/windows_aarch64/os_windows_aarch64.inline.hpp

```diff
@@ -32,4 +32,10 @@ inline bool os::register_code_area(char *low, char *high) {
   return true;
 }
 
+#define HAVE_PLATFORM_PRINT_NATIVE_STACK 1
```

Comment:
As mentioned above perhaps remove the value; also consider adding a #undef after the method